net/libreswan/patches/050-updown-hotplug.patch

```diff
@@ -0,0 +1,8 @@
+--- a/programs/_updown.xfrm/_updown.xfrm.in
++++ b/programs/_updown.xfrm/_updown.xfrm.in
+@@ -1025,3 +1025,5 @@ case "${PLUTO_VERB}" in
+ 	exit 1
+ 	;;
+ esac
++
++/sbin/hotplug-call ipsec
```

Comment:
I think better would be here 
`/sbin/hotplug-call ipsec`

Reply:
Done in https://github.com/openwrt/packages/pull/19233